src/tools/kdb/rm.cpp

```diff
@@ -35,6 +35,13 @@ int RemoveCommand::execute (Cmdline const & cl)
 
 	KeySet conf;
 	Key x = cl.createKey (0);
+
+	if (x.getNamespace () == kdb::ElektraNamespace::CASCADING)
+	{
+		cerr << "Aborting: Specify a namespace for deleting a key." << endl;
+		return 12;
```

Comment:
Just out of curiosity, have we documented this return code `12` anywhere? If not, wouldn't it maybe make sense to define a macro for this error code? What do you think?

Reply:
Ok, I found the documentation for the error codes now. But maybe it still makes sense to create a macro, so we immediately know what the error code is about without checking the man pages.

Reply:
`12` can be different things for different commands, so we'd possibly need more than one macro. It would kind of be like aliases for return codes, and multiple aliases having the same return code. That seems a little overkill IMHO. Am I missing something?

Reply:
Ok, didn't know that we have multiple meanings for the same return code. I see. We can then leave this for now.